docs/user-guide/query/gremlin/gremlin.md

```diff
@@ -1,5 +1,83 @@
-# Gremlin Queries in Gaffer
+# Gremlin in Gaffer
+
+[Gremlin](https://tinkerpop.apache.org/gremlin.html) is a query language that can be used in graph traversals.
+It is a core component of the Apache Tinkerpop library and can allow users to easily express more complex graph queries.
+
+GafferPop is a lightweight Gaffer implementation of the [TinkerPop framework](https://tinkerpop.apache.org/), where TinkerPop methods are delegated to Gaffer graph operations.
 
 !!! warning
-    Gremlin support is currently limited and experimental in Gaffer. This page
-    is a work in progress to capture the current capabilities.
\ No newline at end of file
+    GafferPop is still in development and has some [limitations](gremlin-limits.md).
+    The implementation is basic and its performance is unknown in comparison to using standard Gaffer `OperationChains`.
+
+The addition of Gremlin as query language in Gaffer 2 allows users to represent complex graph queries in a simpler language akin to other querying languages used in traditional and NoSQL databases.
```

Comment:
I don't think we need the version, as there's some confusion around Gaffer2 version 1 and Gaffer2 Version 2 (Gaffer 2.0.0) 😆